google_compute_engine/test_compat.py

```diff
@@ -37,3 +37,9 @@
   import unittest2 as unittest
 
 builtin = 'builtins' if sys.version_info >= (3,) else '__builtin__'
+
+# Import `reload` to re-import modules for testing compat.
+if sys.version_info < (3, 4):
```

Comment:
Can we flip this if statement so that its consistent with the other version checks in this file? Otherwise it makes it more confusing to think about.